vtStorAtaProtocol/Platform/Windows/ProtocolAtaPassThrough.cpp

```diff
@@ -50,7 +50,7 @@ namespace Protocol
             {
                 cEssenseAta1 essense = cEssenseAta1::Reader(Essense);
                 
-                m_DeviceHandle = essense.GetDeviceHandle();
+                m_DeviceHandle = Handle;
```

Comment:
Does it need to store the device handle as a member? What if the IssuePassThroughDirectCommand method takes in the handle as parameter?